tests/pages/MainToolbarPage.ts

```diff
@@ -0,0 +1,261 @@
+import { Page } from '@playwright/test';
+
+export class MainToolbarPage {
+  readonly page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  get layout() {
```

Comment:
Please rename this `layoutSelector`. `layout` has way too many different meanings.

Reply:
Actually, I think that you mention `layoutSelector` not as the query selector for the layout element, but as the layout selection section haha.

Maybe call it `layoutSelection` is a good option too?